runner.py

```diff
@@ -105,6 +110,11 @@ def get_configs(path: Path) -> List[str]:
             params = common_params.copy()
             params.update(params_set.copy())
 
+            if 'workload-size' in params:
+                if params['workload-size'] not in args.workload_size:
+                    continue
+                del params['workload-size']
```

Comment:
It is better here to use dictionary own methods to remove instead of using `del` for removing object. It seems that your addition are consistent with other same cases in this script code. This is out of the scope for the current PR, but it would be better to fix this kind of thing in other PR(s).